scripts/preinst.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
Please consider updating this file according to the upstream `install.sh`

Reply:
If it was necessary to guarantee execution with bash, I fixed that. But if a different execution method was to be used there? Suppose python